### PR TITLE
CODE RUB: Exam Attachment Filler method order

### LINKED ISSUE #608: CODE RUB: Exam Attachment Filler method order

### PR DESCRIPTION
**Pull request by arafat-pro**
Closes #608

### CODE REVIEW

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/ExamAttachments/ExamAttachmentServiceTests.cs [hunk 1]
```diff
@@ -65,11 +65,13 @@ public ExamAttachmentServiceTests()
         private static string GetRandomMessage() => new MnemonicString().GetValue();
 
         private static SqlException GetSqlException() =>
-         (SqlException)FormatterServices.GetUninitializedObject(typeof(SqlException));
+            (SqlException)FormatterServices.GetUninitializedObject(typeof(SqlException));
+
         private static IQueryable<ExamAttachment> CreateRandomExamAttachments() =>
             CreateExamAttachmentFiller(DateTimeOffset.UtcNow)
-                .Create(GetRandomNumber()).AsQueryable();
+                .Create(GetRandomNumber()).AsQueryable();
```
Comment (hassanhabib) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/ExamAttachments/ExamAttachmentServiceTests.cs:72]: Take away the extra space after the semi-colon

Comment (arafat-pro) [reply OtripleS.Web.Api.Tests.Unit/Services/Foundations/ExamAttachments/ExamAttachmentServiceTests.cs]: Thanks for the review! How does anyone can identify such extra spaces on Visual Studio? Is there any extension, theme, or something else?

Comment (ShriHumrudha) [reply OtripleS.Web.Api.Tests.Unit/Services/Foundations/ExamAttachments/ExamAttachmentServiceTests.cs]: Tools -> Options -> Text Editor -> General -> View White Spaces

image

**Comment by ShriHumrudha** [comment]
> Closes #608

Another tip - Name your commit messages with the tags like 'CODE RUB: Removed whitespaces'.

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/ExamAttachments/ExamAttachmentServiceTests.cs (hunk 1 above)
Comment (arafat-pro) [reply OtripleS.Web.Api.Tests.Unit/Services/Foundations/ExamAttachments/ExamAttachmentServiceTests.cs]: Sincere thanks for the guidance! 
I was missing the important keyword - **whitespace**, hence failing to find out the desired solution from the internet!

**Comment by arafat-pro** [comment]
> > Closes #608
> 
> Another tip - Name your commit messages with the tags like 'CODE RUB: Removed whitespaces'.

I'll try my level best to follow this guidance!

**Pull request merged**